app/main.py

```diff
@@ -1 +1,13 @@
-# write your code here
+def copy_file(command: str) -> None:
+    parts = command.split()
+    if len(parts) == 3:
+        _, filename, new_filename = parts
+        if filename != new_filename:
+            try:
+                with (open(filename, "rb") as source,
+                      open(new_filename, "wb") as destination):
+                    destination.write(source.read())
+            except FileNotFoundError:
+                print(f"File not found: {filename}")
+            except Exception as e:
+                print(f"An error occurred: {e}")
```

Comment:
You don't need to catch errors here. You will just have a regular error with regular traceback, it's really fine. You should handle that only if you want to change logic somehow, no need to just rewrite messages